### PR TITLE
docs(datadog_agent source): Fix description rendering

### PR DESCRIPTION
**Pull request by jszwedko**
Component descriptions were being limited in allowed characters. This
relaxes it to just allow strings. I'm unsure if there was a reason we
were restricting this though @binarylogic do you know?

Fixes: #9408 

Signed-off-by: Jesse Szwedko <jesse@szwedko.me>

### CODE REVIEW

File: website/cue/reference/components/sources/datadog_agent.cue [hunk 1]
```diff
@@ -12,11 +12,12 @@ components: sources: datadog_agent: {
 		To send logs from a Datadog Agent to this source, the [Datadog Agent](\(urls.datadog_agent_doc)) configuration
 		must be updated to use:
 
-		```
+		```toml
 		logs_config.dd_url = "<VECTOR_HOST>:<SOURCE_PORT>"
 		logs_config.use_v2_api = false # source does not yet support new v2 API
 		logs_config.use_http = true # this source only supports HTTP/HTTPS
 		logs_config.logs_no_ssl = true|false # should match source SSL configuration.
+		```
```
Comment (spencergilbert) [inline website/cue/reference/components/sources/datadog_agent.cue:20]: 🤔 I thought dd agent was configured with yaml

Comment (jszwedko) [reply website/cue/reference/components/sources/datadog_agent.cue:20]: Hah, yes indeed, the config here is TOML, but it should be YAML. I'll update.

Comment (spencergilbert) [reply website/cue/reference/components/sources/datadog_agent.cue]: WOMP

**Review by spencergilbert — APPROVED** [review]

**Review by spencergilbert — APPROVED** [review]
doh

**Comment by jszwedko** [comment]
I spoke with Ben who mentioned this `description` field should be just a paragraph of plain text such that the agent config fits better in a "How It Works" section

**Pull request merged**